src/components/ui/Text.tsx

```diff
@@ -108,19 +115,34 @@ export function LinkText({
   title?: string
   gradientFrom?: TGradientColorStops
   gradientTo?: TGradientColorStops
+  underline?: boolean
 }) {
   if (internal)
     return (
       <Link
         to={url}
-        className={linkText(small, extraSmall, bold, gradientFrom, gradientTo)}
+        className={linkText(
```

Comment:
Looks like we have two same `className` here and on `L138`, let's put it into `const commonStyles` or something like that

Reply:
fixed, ty